[Bug]: test_vs_chassis_setup.py should not run and fail on non vs_chassis topologies

### Issue Description

Test runs on T2 non vs_chassis when it should only run on vs_chassis

### Results you see

Test runs on T2 non vs_chassis and fails

### Results you expected to see

Test skip on T2 non vs_chassis

### Is it platform specific

generic

### Relevant log output

```shell
=========================== short test summary info ============================
FAILED test_vs_chassis_setup.py::test_midplane - KeyError: 'vs_chassis'
FAILED test_vs_chassis_setup.py::test_chassisdb - KeyError: 'vs_chassis'
================== 2 failed, 2 warnings in 193.37s (0:03:13) ===================
```


### Output of `show version`

```shell
Using 202405
```


### Attach files (if any)

_No response_

--- a/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
+++ b/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml
@@ -1662,6 +1662,15 @@ test_pktgen.py:
       - "https://github.com/sonic-net/sonic-mgmt/issues/13804"
 
 #######################################
+#####         vs_chassis          #####
+#######################################
+test_vs_chassis_setup.py:
+  skip:
+    reason: "Skip vs_chassis setup on non-vs testbeds"
+    conditions:
+      - "asic_type not in ['vs']"
+
+#######################################
 #####            vlan             #####
 #######################################
 vlan/test_vlan.py::test_vlan_tc7_tagged_qinq_switch_on_outer_tag:

